### DISCUSSION TITLE
Not working properly - no bookmark icon, no bookmarks sidebar

### DISCUSSION THREAD

**Discussion by stellarwebworks** [alefragnani/vscode-bookmarks#449] [Q&A] [answered]
I installed the bookmarks plugin (version 13.1.0) in latest version of Visual Studio Code (1.57.1) but it does not seem to be working properly. When I bookmark a line I expect to see a bookmark icon in the gutter but there is none. Also there is no sidebar as shown in the instructions.
It does work partially in that the bookmarks seem to be created and I can see a little blue mark in the scrollbar which I can click on and it takes me to the line that I bookmarked. I tried uninstalling, restarting VS Code and reinstalling but did not help. Please advise.

**Comment by alefragnani** [comment, marked as answer]
> Also posted on `issues` 

Hi @stellarwebworks ,

I have seen the _missing Side Bar icon_ a few times, and up until now, it has always been caused by some issue in the installation process (which is managed by VSCode). It happened not only on this extension, but from other developers too. BTW, I guess two releases ago this was happening a lot for extension developers, while debugging the extensions.

To fix this installation issue, you must reinstall the extension. But you must do a _full reinstall_, which means:

* Go to the Extensions view
* Select the extension
* Click `Uninstall`
* Instead of _reloading_ the window as prompted, you must **Close all VS Code windows**. This will make sure the older installation is fully removed from VS Code internals, once you open VS Code again.
* Open VS Code again
* Go to the Extensions view
* Install the extension again

Please, try these steps to see if it fixes the missing resources.

Hope this helps

**Reply by stellarwebworks** [reply]
responded on [issue 450](https://github.com/alefragnani/vscode-bookmarks/issues/450)
